src/main/openapi/daam.yaml

```diff
@@ -185,8 +185,12 @@ paths:
             schema:
               $ref: "#/components/schemas/CreateApplication"
       responses:
-        "204":
-          description: Successful Response (no content)
+        "201":
```

Comment:
important: 201 - CREATED status requires response header `Location` with the resource URI, and empty body. Either you do it or you retrieve 200 with `CreateApplicationResponse`. I liked 201, but it's up to you.

Reply:
@brunopacheco1 Maybe then lets go with 200 and let the frontend create the application URL.

Because if we take Location header approach, the backend then have to know the resource URL (i.e. URL to access application in frontend) which is not ideal